vue3-apollo-quasar/src/components/ProfilePageLayout/ProfilePageLayout.vue

```diff
@@ -16,19 +16,69 @@
           <UserProfileCard :username="username" />
         </div>
         <!-- Right side -->
-        <div class="tab-contents col" v-if="!loadingLanguage">
+        <div v-if="!!isQuerying" class="tab-contents col">
           <SearchFilter :languages="getLanguages" />
+          <q-separator class="q-mt-sm" />
+          <div
+            class="text-caption q-my-md row justify-between items-center"
+            v-show="!isOnlySorted"
+          >
+            <div class="col">
+              <small class="text-caption text-lowercase">
+                <!-- length of filtered array -->
+                <strong>{{ filteredAndSortedRepos.length }}</strong>
+                results for
+                <!-- repo type -->
+                <strong
+                  v-show="
+                    filterTypeData.filterType &&
+                    filterTypeData.filterType !== defaultFilterType
+                  "
+                >
+                  {{ modifyFilterTypeText(filterTypeData.filterType) }}
+                </strong>
+                repositories
+                <!-- search text -->
+                <span v-show="searchData.search"
+                  >matching <strong> {{ searchData.search }} </strong></span
+                >
+                <!-- Language -->
+                <span
+                  v-show="
+                    languageData.language &&
+                    languageData.language !== defaultLanguage
+                  "
+                >
+                  written in
+                  <strong> {{ languageData.language }} </strong></span
+                >
+                sorted by
+                <!-- Sorted text -->
+                <strong>
+                  {{ sortByData.sortby }}
+                </strong>
+              </small>
+            </div>
+            <div>
+              <a :href="'/' + username" class="row items-center clear_filter">
+                <q-icon
+                  name="fa fa-times"
+                  class="text-white rounded-borders clear_filter_icon q-mx-sm"
```

Comment:
Same issue I mentioned in a different PR. Why do we have different CSS conventions in the same codebase? `clear_filter_icon` looks like the type of value that would be passed as a `name` prop in a `<q-icon>` but not as a CSS class.

Suggestion would be to try and stick to a convention or document convention breaking changes in the README:
eg. Either change `clear_filter_icon` to `clear-filter-icon` or add documentation that specifies something like: all `snake_case` classes are custom classes while all `kebab-case` classes are builtin quasar classes. 

Imo that would get rid of assumptions on the part of the user. 

--
Also, since this issue spans across multiple PRs, I'd recommend creating a PR that standardizes CSS classes used convention wise.